README.md

```diff
@@ -8,15 +8,21 @@ Crypto and security extensions to OpenSAML
 
 ---
 
-**NOTE:** OpenSAML has introduced support for ECDH key agreement starting from version 4.1.0. Therefore, this functionality has been removed from the opensaml-security-ext library in version 3.0.0. If you are using OpenSAML 4.0.X you need to use version 2.1.0 of opensaml-security-ext and 1.0.7 of you are still using OpenSAML 3.
+**NOTE:** OpenSAML has introduced support for ECDH key agreement starting from version 4.1.0. Therefore, this functionality has been removed from the opensaml-security-ext library in version 3.0.0. If you are using OpenSAML 4.0.X you need to use version 2.1.0 of opensaml-security-ext and 1.0.8 of you are still using OpenSAML 3.
 
 ---
 
-The opensaml-security-ext extends the core OpenSAML libraries with the capability to encrypt and decrypt XML data using ephemeral-static ECDH key agreement (pre-3.0 only). 
+**NOTE:** There is an interoperability issue between OpenSAML 4.0.X and version 1.0.7 with regard to the use of ECDH. The issue conerns the requirements on precense of ConcatKDF parameters in ECDH where the interpretation of the standard differes and where OpenSAML does not allow the empty parameters sent by 1.0.7. An issue is raised with the OpenSAML team to resolve this issue to bring OpenSAML 4 in alignment with 1.0.7 if possible.
```

Comment:
Some spelling errors: 
"The issue conerns" -> "The issue concerns"
"precense" -> "presence"
"differes" -> "differs"